lib/puppet/provider/package/pip.rb

```diff
@@ -29,7 +29,11 @@ def self.parse(line)
   # that's managed by `pip` or an empty array if `pip` is not available.
   def self.instances
     packages = []
-    pip_cmd = which(cmd) or return []
+    if cmd.class == Array
```

Comment:
To cover both cases having either /usr/bin/pip or /usr/bin/pip-python as the binaries for pip provider.
Because currently the latest and the only available python-pip package in EPEL repos for EL6 provides the "/usr/bin/pip" binary, whereas the older package was providing "/usr/bin/pip-python" binary.
make sense ?